db/schema.rb

```diff
@@ -43,6 +43,7 @@
     t.integer  "customer_id"
     t.integer  "c2_status",       default: 0,    null: false
     t.string   "token"
+    t.datetime "archived_at"
```

Comment:
I would default to not adding a new DB column when we don't need it. If we added something like `papertrail`, it would keep track of all data changes in the case that we needed them later